addon/src/main/java/fi/jasoft/dragdroplayouts/client/ui/panel/VDDPanel.java

```diff
@@ -170,7 +170,9 @@ protected void updateDragDetails(VDragEvent event) {
         Element over = event.getElementOver();
 
         Widget content = Util.findWidget(over, null);
-        VConsole.log(content.getElement().toString());
+        if (content != null) {
```

Comment:
Maybe just remove this log statement totally? I don't see why we would want to fill the debug console with this.